src/stable/lib/state.ts

```diff
@@ -189,17 +189,23 @@ globalThis._azleDispatch = (action: Action): void => {
     }
 
     if (action.type === 'DELETE_AZLE_REJECT_CALLBACK') {
-        delete globalThis._azleRejectCallbacks[action.payload];
+        if (globalThis._azleRejectCallbacks) {
+            delete globalThis._azleRejectCallbacks[action.payload];
+        }
         return;
     }
 
     if (action.type === 'DELETE_AZLE_RESOLVE_CALLBACK') {
```

Comment:
I feel like we should throw in these cases, in the DELETE_ action cases, if the global variables are not defined.